src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -30,7 +30,6 @@ const ConstrainedHeightFlex = styled(Flex)`
     height: calc(100vh - 55px);
   `}
   & > * {
-    overflow-y: scroll;
```

Comment:
Didn't understand 👆. was this change intentional?

Reply:
oh no